cmake/CMakeLists.txt

```diff
@@ -310,10 +310,17 @@ if(PKG_VORONOI)
   option(DOWNLOAD_VORO "Download voro++ (instead of using the system's one)" OFF)
   if(DOWNLOAD_VORO)
     include(ExternalProject)
+
+    if(BUILD_SHARED_LIBS)
+        set(VORO_BUILD_OPTIONS "CFLAGS=-fPIC")
+    else()
+        set(VORO_BUILD_OPTIONS "")
```

Comment:
The `else()` part could just be neglected or at least `set(VORO_BUILD_OPTIONS)`.